src/gracetabnote.ts

```diff
@@ -8,7 +8,7 @@
 //
 // See `tests/gracetabnote_tests.ts` for usage examples.
 
-import { Font } from './font';
+// import { Font } from './font'; // RONYEH
```

Comment:
You could leave this file unmodified, `Font.SANS_SERIF`is already merged producing no visual diff.